registry.json

```diff
@@ -598,6 +598,20 @@
  "telegram": "ADA_Pilot",                                            
  "operatorName": "Steven",                                
  "github": "StevieMac415" 
+},
+{  
+ "ticker": "BZH",                                                      
+ "poolId": "9b9ad921921db31ca91cd6dfdf11f5efee8c1ab94671a6b4a8edc748", 
+ "social": {                                                           
+   "twitter": "BreizhPool",
+   "website": "https://breizhstakepool.io",
+   "facebook": "",  
+   "instagram": "",
+   "youtube": ""
+ },
+ "telegram": "bzhsp",                                            
```

Comment:
Please use your personal telegram handle.

Reply:
I did the changes.. my personal telegram handle is YannLgrnd